quickstarts/dojo/config.yml

```diff
@@ -0,0 +1,46 @@
+slug: dojo
+
+description: |
+  ## Comprehensive monitoring quickstart for Dojo
+  Monitor your Dojo application performance in the most complex and distributed environments to deliver exceptional user experience by proactively identifying and resolving performance issues without any downtime.
+
+  ## Why monitor Dojo?
+  The performance monitoring tool for Dojo application helps you in monitoring the entire infrastructure and identifies the critical issues in your application. Dojo performance monitoring enables you to get a complete picture of Dojo errors, as well as know when and where your Dojo application slows down. 
```

Comment:
This sentence doesn't really tell me what Dojo does. What am I monitoring with this? What type of application is it? From the doc, I gather that these are web applications. From the https://dojo.io/learn/overview/introduction website, I gather that this is a tool for developing web-based applications. At the very least, we should reference this.

I'd rewrite this as something like:
"As you know, you Dojo web app is a complex interaction of widgets, data flows, and data stores. By monitoring your Dojo app with New Relic, you can start to gain insight into its performance and areas where it can be improved."